package.json

```
@@ -25,30 +25,30 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "@babel/eslint-parser": "^7.25.1",
+    "@babel/core": "^7.27.4",
+    "@babel/eslint-parser": "^7.27.5",
     "@redocly/cli": "^1.34.3",
     "eslint": "^8.57.1",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^8.10.0",
     "eslint-formatter-gitlab": "^3.0.0",
     "eslint-plugin-html": "^6.2.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
-    "eslint-plugin-security": "^1.5.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
+    "eslint-plugin-security": "^1.7.1",
     "eslint-plugin-sonarjs": "^0.16.0",
-    "eslint-plugin-unicorn": "^45.0.1",
+    "eslint-plugin-unicorn": "^45.0.2",
     "eslint-plugin-workspaces": "^0.7.0",
-    "eslint-plugin-yml": "^1.2.0",
+    "eslint-plugin-yml": "^1.18.0",
     "hbs-cli": "^1.4.1",
     "license-checker": "^25.0.1",
-    "minimist": "^1.2.2",
-    "newman": "^6.1.2",
+    "minimist": "^1.2.8",
+    "newman": "^6.2.1",
     "sinon": "^17.0.1",
-    "typescript": "^5.5.4"
+    "typescript": "^5.8.3"
   },
   "dependencies": {
-    "ajv": "4.11.8 - 8",
+    "ajv": "4.11.8",
```

I think this needs to go back to the following because you've now fixed it at 4.11.8 where previously it was allowing any version between 4.11.8 and version 8 so some of the dependencies in pnpm-lock.yaml have now been downgraded from 6.x down to 4.11.8
```suggestion
    "ajv": "4.11.8 - 8",
```